axlearn/common/compiler_options.py

```diff
@@ -55,11 +110,20 @@ def default_xla_options(
             xla_tpu_data_parallel_opt_different_sized_ops="true",
             # Group non-blocking DCN collectives into as few stages as possible.
             xla_tpu_enable_sunk_dcn_allreduce_done_with_host_reduction="true",
+            # Change to 16GB. The default is 4GB which is too small for larger models. This
+            # cause the step time to be double. You should increase this
+            # further if you see "Allocator failed to allocate". A feature
+            # to dynamically allocate may come later: b/380514965
+            megascale_grpc_premap_memory_bytes=17179869184,
```

Comment:
This will affect all jobs. Is it a desired change? Should we limit it to v6e only?

Reply:
I think this is important for all jobs. The only downside is that it takes host memory.

Reply:
Done, but I do still recommend enabling this for all jobs. Let's tackle that in separate PR.